CHANGELOG.rst

```diff
@@ -1,22 +1,26 @@
+1.0.0 (April 7th, 2017)
```

Comment:
Apparently this was merge to master directly. @88manpreet what are we supposed to do with this pr now?